modules/notification/notification.go

```diff
@@ -26,7 +27,9 @@ func RegisterNotifier(notifier base.Notifier) {
 
 func init() {
 	RegisterNotifier(ui.NewNotifier())
-	RegisterNotifier(mail.NewNotifier())
+	if setting.Service.EnableNotifyMail {
```

Comment:
This must be changed.

Reply:
I call this initialization from routers/init.go now.

Reply:
New solution ok?